timemachine/fe/rbfe.py

```diff
@@ -45,11 +48,19 @@ def setup_in_vacuum(st, ligand_conf, lamb):
     return x0, box0, hmr_masses, potentials, baro
 
 
-def setup_in_env(st, host, host_config, ligand_conf, lamb, temperature, run_seed):
+def setup_in_env(
+    st: SingleTopology,
+    host_system: VacuumSystem,
+    host_masses: List[float],
+    host_conf: NDArray,
+    host_config: HostConfig,
```

Comment:
nit: Seems kind of odd to pass in host_config just to return host_config.box.

Reply:
Good call, cleaned up in https://github.com/proteneer/timemachine/pull/959/commits/6b336d5221876ea4679339f811b6e0e8bafa7768